pvactools/lib/aggregate_all_epitopes.py

```diff
@@ -452,6 +455,17 @@ def get_tier(self, mutation, vaf_clonal):
             if mutation["Transcript Support Level"] > self.maximum_transcript_support_level:
                 tsl_pass = False
 
+        ic50_pass = mutation["{} MT IC50 Score".format(self.mt_top_score_metric)] < binding_threshold
+        percentile_pass = (
+            self.percentile_threshold is not None and 
+            mutation["{} MT Percentile".format(self.mt_top_score_metric)] < self.percentile_threshold
+        )
+        thresholds_passed = (
```

Comment:
This is just for ease of reading the code but I would name this variable `binding_pass` since some of the other criteria (e.g. allele expr) also have thresholds/cutoffs.